solutions/usaco-990.mdx

```diff
@@ -212,4 +212,48 @@ public class berries {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+import sys
+
+sys.stdin = open("berries.in", "r")
+sys.stdout = open("berries.out", "w")
+
+input = sys.stdin.readline
```

Comment:
why. you already synced the io

Reply:
i can change it if you want but it's correct? using sys.stdin.readline instead of just input is faster. & i thought using using sys.stdin and sys.stdout would be neater and just more simple but it seems like my opinion is the minority so i'll change it

Reply:
hmm, that makes sense. i think it might be a little bit more confusing (especially because you don't need it to pass), so it might be better to remove it.

it's completely up to you whether you want to keep it though!